matrix_is_tester/is_api.py

```diff
@@ -34,9 +34,10 @@ class IsApi(object):
     def __init__(self, base_url, version, mail_sink):
         """
         Args:
-            base_url: The base URL of the IS API to use
-            version: Version of the IS API (eg. 'v1' or 'v2')
-            mail_sink: Mail sink object to use for getting email authentication tokens.
+            base_url (bytes): The base URL of the IS API to use
```

Comment:
are you sure? you seem to be passing `str`s in and then using them as `str`s in the function.

... which, incidentally, isn't great practice, since it will do different things on py2 and py3. Sorry, I should have picked this up sooner: you need to decide whether you have bytes or unicode and, for code that supports both python versions, prefix string constants with `b` or `u` accordingly.  

On the other hand, I'm not sure either of us is going to survive the week if I now block this PR on sorting out the unicode correctness of all your strings, so please can you just fix up this docstring to say `str` and file the rest under "things to fix one day".

Reply:
yeah, fair enough. Have changed and XXXed.